src/utilities/cnpj/index.ts

```diff
@@ -27,8 +27,12 @@ export const FIRST_CHECK_DIGIT_WEIGHTS = [5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2];
 
 export const SECOND_CHECK_DIGIT_WEIGHTS = [6, ...FIRST_CHECK_DIGIT_WEIGHTS];
 
-export function format(cnpj: string): string {
-  const digits = onlyNumbers(cnpj);
+export function format(cnpj: string | number, fillZeroes: boolean = false): string {
```

Comment:
Instead of just a boolean parameter, I think that is better to use an object. If in the future we need another `config` we can just add without a break the API.

Reply:
Great! But in this new format fillZeroes seems a little longer...
```js
format('', { fillZeroes: true })
```
Maybe just `pad` is better (and looks more standard):
```js
format('', { pad: true })
```

Reply:
Sounds good to me.

Reply:
Utilities, tests and docs updated!